lib/slax_web/websockets/issue.ex

```diff
@@ -115,14 +84,14 @@ defmodule SlaxWeb.Issue do
     end
   end
 
-  defp labels_for_issue(issue) do
+  def labels_for_issue(issue) do
     column_label =
       cond do
         not is_nil(issue["closed_at"]) ->
           ["closed"]
 
-        Enum.empty?(issue["labels"]) ->
-          ["icebox"]
+        issue["draft"] == true ->
+          ["draft"]
```

Comment:
I've never seen "icebox" as a useful label, but knowing if a PR is a draft could be.